### DISCUSSION TITLE
清爽首页的番剧时间表建议

### DISCUSSION THREAD

**Discussion by LockRim** [the1812/Bilibili-Evolved#3148] [功能建议 / Ideas]
v1版的竖置时间表或许会更好一些
v2的横屏数据多但也更加密集了
不知道是不是我的问题，无法直接通过时间表点进番剧

**Comment by the1812** [comment]
一般看几天的就够了? 我看有两周的数据就分成一屏 7 天了

**Reply by LockRim** [reply]
7天就足够了，作者大大辛苦

**Reply by the1812** [reply]
现在就是7天啊，不是说太密集了吗

**Reply by LockRim** [reply]
每行之间加些空间？这样5天大概就够了
或者是v1的竖着排列，横着排列总感觉参差不齐（看习惯了倒也还好）
